src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -325,7 +325,19 @@ export default class ActionCommand extends ResponsiveSlashCommandSubcommandBuild
       (member: GuildMember) => Promise<boolean>,
       (member: GuildMember, reason: string, days?: number) => Promise<boolean>,
       ExtraActionOptions
-    ]
+    ],
+    [
+      APIApplicationCommandOptionChoice<string>,
+      (member: GuildMember) => Promise<boolean>,
+      (member: GuildMember, reason: string) => Promise<boolean>,
```

Comment:
not sure if reason here and on line 338 are necessary

Reply:
its used in /verify as well, unsure why but ill keep it in to keep up with the code skyler made